client/error.go

```diff
@@ -94,6 +97,9 @@ func HandleErrors(err error) {
 	if httpStatusCode == 500 {
 		headline = "An internal error occurred."
 		subtext = details
+	} else if httpStatusCode == 503 {
```

Comment:
Not in the scope of this PR, but maybe in the future we'll start using the built-in status code constants
https://golang.org/src/net/http/status.go

Reply:
Done